src/pages/SubmissionDetail.vue

```diff
@@ -7,72 +7,74 @@
           <p>Model Name: {{model_name}}</p>
           <p>Status: {{status}}</p>
           <p>Submitted At: {{submitted_at}}</p>
+          <p v-if="error">Error Message: {{error_msg}}</p>
         </card>
       </div>
     </div>
 
-    <div class="row">
-      <div class="col-md-6">
-        <card>
-          <h4 class="card-title">Robustness Score</h4><hr>
-          <h2 style="text-align:center">{{data.robustness}}</h2>
-        </card>
-      </div>
-      
-      <div class="col-md-6">
-        <card>
-          <h4 class="card-title">Rating</h4><hr>
-          <h2 style="text-align:center">{{data.rating}}</h2>
-        </card>
+    <div v-if="!error">
+      <div class="row">
+        <div class="col-md-6">
+          <card>
+            <h4 class="card-title">Robustness Score</h4><hr>
+            <h2 style="text-align:center">{{feedback.robustness}}</h2>
+          </card>
+        </div>
+        
+        <div class="col-md-6">
+          <card>
+            <h4 class="card-title">Rating</h4><hr>
+            <h2 style="text-align:center">{{feedback.rating}}</h2>
+          </card>
+        </div>
       </div>
-    </div>
 
-    <div class="row">
-      <div class="col-md-12">
-        <card>
-          <h4 class="card-title">Details</h4><hr>
-          <div slot="raw-content" class="table-responsive">
-            <table class="table">
-              <thead>
-                <th v-for="column in tableHeaders">{{column}}</th>
-              </thead>
-              <tr>
-                <td v-for="column in tableAccuracies">{{column}}</td>
-              </tr>
-              <tr>
-                <td v-for="column in tableConfidences">{{column}}</td>
-              </tr>
-            </table>
-          </div>
-        </card>
+      <div class="row">
+        <div class="col-md-12">
+          <card>
+            <h4 class="card-title">Details</h4><hr>
+            <div slot="raw-content" class="table-responsive">
+              <table class="table">
+                <thead>
+                  <th v-for="column in tableHeaders">{{column}}</th>
+                </thead>
+                <tr>
+                  <td v-for="column in tableAccuracies">{{column}}</td>
+                </tr>
+                <tr>
+                  <td v-for="column in tableConfidences">{{column}}</td>
+                </tr>
+              </table>
+            </div>
+          </card>
+        </div>
       </div>
-    </div>
 
-    <div class="row">
-      <div class="col-12">
-        <chart-card title="Accuracy Plot"
-                    chart-type="Bar"
-                    :chart-data="chartData.data"
-                    :chart-options="chartData.options">
-        </chart-card>
-      </div>
+      <div class="row">
```

Comment:
Do we need to let the user choose chart or table later on?

Reply:
Yeah. Will do this later.